sdcm/cluster.py

```diff
@@ -5647,6 +5648,45 @@ def download_scylla_monitoring(self, node):
         if node.distro.is_ubuntu:
             node.remoter.run(f'sed -i "s/python3/python3.6/g" {self.monitor_install_path}/*.py')
 
+    @property
+    def monitoring_template(self) -> Path:
+        return Path(get_data_dir_path("monitoring-dash-template.json"))
+
+    def configure_overview_template(self, node: BaseNode):
+        def find_overview_row(row):
+            return row["class"] == "row" and len(row.get("panels", [])) > 0 and row["panels"][0].get("class", "") == "alert_table"
+
+        template_file = f"/tmp/scylla-overview.template.{node.name}.json"
```

Comment:
for adjusting remote files like that we actually have ctx mgr: `remote_file` did you check if could be reused?

Reply:
Fixed